.travis.yml

```diff
@@ -4,14 +4,23 @@ go:
   - 1.13.x
   - tip
 
+cache:
```

Comment:
Main reason for using cache is to speed up things that don't need to be repeated. Of the two things cached here, `$GOPATH/pkg/mod` is the most helpful -- it will prevent dependent modules from being downloaded/fetched on every verification, which I do think contributes non-trivial time (I can compare with and without and post if that would be helpful).

The build cache should speed up builds in cases where certain portions of the code haven't changed, but is not as important since a cache miss doesn't require going over the network. Could make sense to omit for simplicity.